app/forms/disk-attach.tsx

```diff
@@ -57,6 +57,7 @@ export function AttachDiskSideModalForm({
       <ComboboxField
         label="Disk name"
         placeholder="Select a disk"
+        description=""
```

Comment:
?

Reply:
Ah, sorry; was from an earlier experiment. Not necessary because of better defaults in ComboboxField. Will get this and 2 other spots.